mobile-widgets/qml/DivePlannerSetup.qml

```diff
@@ -245,7 +245,7 @@ TemplatePage {
 				}
 				TemplateSpinBox {
 					from: 1
-					to: 99
```

Comment:
There can be situations where values above 100 are desirable. I would allow at least 120 if not 200.

Reply:
That's interesting - I have so far only ever come across Gradient Factors being used in the range of 1 - 100%. Do you have references for literature on the use of Gradient Factors above 100%

Reply:
Of course, that would be violating plain vanilla Bühlmann rules. But there are other models. For example, some VPM-B schedules, even with +2 conservatism can end up in the range of GF-high about 120. So, when you follow those you might want to compare to Bühlmann+GF and then you need such values. Not that I would recommmend diving those schedules... 

Literature? I could offer https://thetheoreticaldiver.org/wordpress/index.php/2017/11/02/vpm-b-gradients-as-gradient-factors/

Reply:
> Of course, that would be violating plain vanilla Bühlmann rules.

To my understanding the 'Bühlmann rules' really just denote that at GF 100/100 the risk of getting DCI was what Bühlmann considered to be acceptable. So it is entirely possible to think of situations where using a decompression schedule that has got a higher risk than this is the best possible course of action - like a contingency where there isn't enough gas available to complete decompression at GF 100/100. But then of course it is not likely that it will be possible to use Subsurface to create a decompression plan when such a situation eventuates...

> Literature? I could offer [thetheoreticaldiver.org/wordpress/index.php/2017/11/02/vpm-b-gradients-as-gradient-factors](https://thetheoreticaldiver.org/wordpress/index.php/2017/11/02/vpm-b-gradients-as-gradient-factors/)

Interesting article - I know next to nothing about how VPM-B works, so this was educational. By the way the link in the first paragraph seems to be broken.